client/src/components/PreviewImageField/PreviewImageField.js

```diff
@@ -296,13 +309,19 @@ class PreviewImageField extends Component {
     ) : null;
     let messageBox = null;
 
-    if (message) {
+    if (errors || status === 'error') {
+      const errorMessage = errors && errors.value
+      ? errors.value
+      : i18n._t('AssetAdmin.DROPZONE_RESPONSE_ERROR', 'Server responded with an error.');
```

Comment:
```suggestion
        ? errors.value
        : i18n._t('AssetAdmin.DROPZONE_RESPONSE_ERROR', 'Server responded with an error.');
```
Indent these to make it easier to scan-read - they're part of the same const assignment as the line above.